### DISCUSSION TITLE
No masks found thing

### DISCUSSION THREAD

**Discussion by Lchaxlover** [winstxnhdw/lc-hax#485] [Q&A]
You guys have said that the only mob you can "spawn" in other peoples games is a mask but when i type the command it says no masks found. Why is this?

**Comment by winstxnhdw** [comment]
You need the mask item.

**Reply by Lchaxlover** [reply]
Is there a way to give myself a mask

**Reply by Lchaxlover** [reply]
did u just make that?

**Reply by Lchaxlover** [reply]
Wait does it create an item or just take one from somewhere on the map

**Reply by winstxnhdw** [reply]
Takes one from somewhere. No, it’s been here for a while.

**Reply by Lchaxlover** [reply]
So if theres no mask in the a game you cant spawn them in

**Reply by Lchaxlover** [reply]
You should make it so you can make the item

**Reply by winstxnhdw** [reply]
Why would I not do that if I could.

**Reply by Lchaxlover** [reply]
Good point
